packages/components-js/package.json

```diff
@@ -75,18 +75,18 @@
     "@playwright/test": "1.27.0",
     "@porsche-design-system/shared": "0.0.0",
     "@types/js-beautify": "^1.13.3",
-    "@types/node": "^18.11.18",
+    "@types/node": "^18.14.5",
     "concurrently": "^7.6.0",
-    "devtools-protocol": "^0.0.1087818",
+    "devtools-protocol": "^0.0.1112051",
     "globby": "^11.0.4",
     "js-beautify": "^1.14.7",
     "jsdom": "^20.0.3",
-    "local-web-server": "^5.2.1",
+    "local-web-server": "^5.3.0",
     "terser-webpack-plugin": "^5.3.6",
     "ts-node": "^10.9.1",
-    "typescript": "~4.9.4",
+    "typescript": "~4.9.5",
```

Comment:
I noticed that we have `"typescript": "~4.6.4"` in `/packages/components-angular/package.json` - should we also update it there?

Reply:
Angular decides which typescript versions are allowed, depending on the version of angular we use. Thats why its pinned. See dependencies.md.